src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -1294,6 +1298,12 @@ public override BoundNode VisitEventAssignmentOperator(BoundEventAssignmentOpera
         }
 
         protected virtual void VisitArguments(ImmutableArray<BoundExpression> arguments, ImmutableArray<RefKind> refKindsOpt, MethodSymbol method)
+        {
+            VisitArgumentsBeforeCall(arguments, refKindsOpt);
```

Comment:
Is this method just for non-method calls that take arguments? indexers, etc.

Reply:
It's also used by indexers. I don't see why that matters though?

Reply:
just checking my understanding :)